internal/packaging/lifecycle_other.go

```
@@ -8,6 +8,34 @@ import (
 	"os/exec"
 )
 
+// runSystemctlUser runs `systemctl --user <verb> <ServiceLabel>.service`
+// and wraps a non-zero exit with the combined output. Shared by
+// stopForOS / startForOS / restartForOS so the verb-specific
+// boilerplate doesn't duplicate three ways (SonarCloud per-PR
+// duplication gate caught this).
+func runSystemctlUser(verb string) error {
+	out, err := exec.Command("systemctl", systemdUserFlag, verb, ServiceLabel+systemdUnitSuffix).CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("systemctl --user %s: %v: %s", verb, err, string(out))
```

golangci-lint (2.12.2)

[error] non-wrapping format verb for fmt.Errorf. Use `%w` to format errors

(errorlint)